### ISSUE TITLE
Attribute type usage. Missing explicit disambiguation qualification on name clash.

### ISSUE THREAD

**Issue by tamlin-mike** [icsharpcode/ILSpy#1149]
Compilable code:
```
using NS1;
using NS2;
namespace NS1 { public class Tst1Attribute : System.Attribute {} }
namespace NS2 { public class Tst1Attribute : System.Attribute {} }
class C {
 [NS2.Tst1]
 void foo() {}
}
```
decompiles into
```
...
class C {
 [Tst1] // <-- Missing required explicit namespace disambiguation
 void foo() {}
}
```

I suspect the missing disambiguation might be due to ILSpy is checking for the name "Tst1", not "Tst1Attribute". If so, this should be an easy fix.

**Comment by siegfriedpammer** [comment]
See https://github.com/icsharpcode/ILSpy/blob/b3db47321119b8324d8851c089e01d66fbafffde/ICSharpCode.Decompiler/CSharp/Transforms/IntroduceUsingDeclarations.cs#L178-L189

You are welcome to step in and provide a fix! :)

If you need any help, please ask!

**Comment by siegfriedpammer** [comment]
Should be fixed in c7c3801ff12a503a18dd450a88bb80aaea884676
